src/Error/ExceptionRenderer.php

```diff
@@ -183,7 +183,6 @@ public function render()
                 'format' => 'array',
                 'args' => false
             ]);
-            $viewVars['_serialize'][] = 'trace';
```

Comment:
I like them too, however xml bugged out.
Should I add the info to the book @ logs in this case?